Modules/HypnokinkModule/Characters/HK_Rush.gd

```diff
@@ -0,0 +1,150 @@
+		InterestTopic.FeminineBody: Interest.KindaLikes,
+		InterestTopic.AndroBody: Interest.Likes,
+		InterestTopic.MasculineBody: Interest.KindaLikes,
+		InterestTopic.ThickBody: Interest.KindaLikes,
+		InterestTopic.AverageMassBody: Interest.Likes,
+		InterestTopic.SlimBody: Interest.ReallyLikes,
+		InterestTopic.ThickButt: Interest.ReallyLikes,
+		InterestTopic.AverageButt: Interest.Likes,
+		InterestTopic.SlimButt: Interest.KindaLikes,
+		InterestTopic.NoBreasts: Interest.Likes,
+		InterestTopic.MediumBreasts: Interest.Dislikes,
+		InterestTopic.BigBreasts: Interest.ReallyDislikes,
+		InterestTopic.LactatingBreasts: Interest.ReallyDislikes,
+		InterestTopic.StuffedPussy: Interest.Dislikes,
+		InterestTopic.StuffedAss: Interest.Likes,
+		InterestTopic.Pregnant: Interest.Dislikes,
+		InterestTopic.StuffedThroat: Interest.KindaLikes,
+		InterestTopic.CoveredInCum: Interest.KindaLikes,
+		InterestTopic.CoveredInLotsOfCum: Interest.SlightlyDislikes,
+		InterestTopic.FullyNaked: Interest.Likes,
+		InterestTopic.ExposedPussy: Interest.Dislikes,
+		InterestTopic.ExposedAnus: Interest.Loves,
+		InterestTopic.ExposedCock: Interest.ReallyLikes,
+		InterestTopic.ExposedPanties: Interest.KindaLikes,
+		InterestTopic.ExposedBra: Interest.Dislikes,
+		InterestTopic.LooseAnus: Interest.Loves,
+		InterestTopic.LoosePussy: Interest.SlightlyDislikes,
+		InterestTopic.TightAnus: Interest.KindaLikes,
+		InterestTopic.TightPussy: Interest.SlightlyDislikes,
+		InterestTopic.NoVagina: Interest.Loves,
+		InterestTopic.HasVaginaOnly: Interest.Hates,
+		InterestTopic.BigCock: Interest.SlightlyDislikes,
+		InterestTopic.AverageCock: Interest.KindaLikes,
+		InterestTopic.SmallCock: Interest.ReallyLikes,
+		InterestTopic.NoCock: Interest.Hates,
+		InterestTopic.HasCockOnly: Interest.Loves,
+	}
+	
+func _getName():
+	return "Rush"
+
+func getGender():
+	return Gender.Male
+
+func getChatColor():
+	return '#F46670'
+	
+func getSmallDescription() -> String:
+	return "A towering stallion."
+	
+func getFightIntro(_battleName):
+	return "Rush towers over you in a textbook boxing stance."
+
+func getSpecies():
+	return ["equine", "dragon"]
+	
+func _getAttacks():
+	return ["biteattack", "simplekickattack", "AttackFury", "DestroyRestraintsAttack", "EvasiveManeuver", "FeralAssault", "trygetupattack"]
+	
+func getThickness() -> int:
+	return 25
+
+func getFemininity() -> int:
+	return 0
+
+func createBodyparts():
+	giveBodypartUnlessSame(GlobalRegistry.createBodypart("horseheadbig"))
+	giveBodypartUnlessSame(GlobalRegistry.createBodypart("mohawkhair"))
+	giveBodypartUnlessSame(GlobalRegistry.createBodypart("horseears"))
+	giveBodypartUnlessSame(GlobalRegistry.createBodypart("dragonhorns2"))
+	giveBodypartUnlessSame(GlobalRegistry.createBodypart("anthrobody"))
+	giveBodypartUnlessSame(GlobalRegistry.createBodypart("buffarms"))
+	giveBodypartUnlessSame(GlobalRegistry.createBodypart("hoofs"))
+	var penis = GlobalRegistry.createBodypart("equinepenis")
+	penis.lengthCM = 35
```

Comment:
It clips through during the animations, are you sure? x3

Reply:
Oversized genitalia ain't really my thing, so I've just picked what seemed a sufficiently ridiculous size. Dropped it a bit.